lib/drivers/tandemDriver.js

```diff
@@ -1221,8 +1230,13 @@ module.exports = function (config) {
 
     function iterate(err, result) {
       if (err) {
-        debug('error retrieving record ', result);
-        callback(err, null);
+        if(err.name === 'multi-fail') {
```

Comment:
Do you assign `name`s to errors elsewhere in this driver or in other drivers? Just wondering if it's a standard. It seems pretty much as arbitrary as matching the `error.message` to me, although certainly more concise.